app/src/main/java/com/yacgroup/yacguide/AscendActivity.kt

```diff
@@ -58,17 +59,22 @@ class AscendActivity : AppCompatActivity() {
 
         // The database query is different from null,
         // if this activity is called from the TourbookActivity or TourbookAscendActivity.
```

Comment:
This comment is not correct. This activity is only called from `TourbookAscendActivity`. right?

Reply:
Correct but this comment is actually from you :)

Reply:
OK, I fixed it.